bench-tps/src/bench_tps_client.rs

```diff
@@ -93,6 +94,16 @@ pub trait BenchTpsClient {
     ) -> Result<Account>;
 
     fn get_multiple_accounts(&self, pubkeys: &[Pubkey]) -> Result<Vec<Option<Account>>>;
+
+    fn get_slot(&self) -> Result<Slot>;
+
+    fn get_blocks(&self, start_slot: Slot, end_slot: Option<Slot>) -> Result<Vec<Slot>>;
+
+    fn get_block_with_config(
+        &self,
+        slot: Slot,
+        rpc_block_config: RpcBlockConfig,
+    ) -> Result<UiConfirmedBlock>;
```

Comment:
if you pass in an `RpcBlockConfig` with a `CommitmentLevel` different than `None` or `Confirmed` is that going to cause any issues here? I assume no since that is built into the RPC server but just want to confirm

Reply:
inside this `RpcBlockConfig` there is a commitment specified, so I think it is consistent.

Reply:
^ realizing now the comment kind of contradicts the function name. Function name says it returns block based on a specific config. but comment says it returns a confirmed block

Reply:
I use confirmed explicitly there. But the method looks confusing also because it returns UiConfirmedBlock

Reply:
cool makes sense. ya that RPC function description may need an update lol